docs/uperf.md

```diff
@@ -53,6 +55,8 @@ $ oc adm policy add-scc-to-user privileged -z benchmark-operator
 
 `pin_client` what node to pin the client pod to.
 
+`multus` Configure our pods to use multus.
```

Comment:
Minor nit but can we throw a link to a multus doc page or something in case someone has no idea what it is?